src/main/steps/respondent/hub-page/latest-update-content/final-order-requested.njk

```diff
@@ -1,3 +1,8 @@
-{% for property, line in finalOrderRequested %}
-  <p class="govuk-body">{{ line }}</p>
-{% endfor %}
+{% if userCase.applicant2AppliedForFinalOrderFirst === 'Yes' %}
+    <p class="govuk-body">{{ finalOrderRequested.line2 }}</p>
+    <p class="govuk-body">{{ finalOrderRequested.line3 }}
+    <a href="https://www.gov.uk/get-help-with-court-fees"
```

Comment:
might be best to have the link and text in the content file as we do that in other places.

Reply:
Moved logic to content file as suggested